Project-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Card.css';
+
+function CardContent() {
+    return (<div className="card-content-container">
```

Comment:
formatting code this way helps keep the indentation look like HTML.
return (
   <div>
       .....
   </div>
)